builtin/logical/aws/path_config_root.go

```diff
@@ -50,6 +51,35 @@ func pathConfigRoot(b *backend) *framework.Path {
 	}
 }
 
+func (b *backend) pathConfigRootRead(ctx context.Context, req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
+	b.clientMutex.RLock()
+	defer b.clientMutex.RUnlock()
+
+	entry, err := req.Storage.Get(ctx, "config/root")
+	if err != nil {
+		return nil, err
+	}
+	var config rootConfig
+
+	if entry != nil {
+		if err := entry.DecodeJSON(&config); err != nil {
+			return nil, err
+		}
+	}
+
+	configData := map[string]interface{}{
+		"access_key":   config.AccessKey,
+		"secret_key":   "",
```

Comment:
Simply omitting the key entirely is the more common pattern.

Reply:
I was mostly worried about people who expect to get the same keys back on reading data as they input in writing to the endpoints, but if this is the more common pattern, makes sense to just omit it. Done.